### DISCUSSION TITLE
Can i use this screen NV3041A with moddable sdk?

### DISCUSSION THREAD

**Discussion by haikyuu** [Moddable-OpenSource/moddable#1412] [Q&A] [answered]
NV3041A—480 RGB x 272 dot TFT LCD Single Chip Driver
Is it supported? https://admin.osptek.com/uploads/NV_3041_A_Datasheet_V1_2_20221011_686486a221.pdf

It comes included in this board
https://a.aliexpress.com/_oCJjPuJ

**Comment by phoddie** [comment, marked as answer]
Probably. It looks like the display controller implements the common MIPI command set. MIPI is used by ILI9341 driver ([docs here](https://moddable.com/documentation/drivers/ili9341/ili9341)). You'll need to define the SPI pins and initial registers for the display configuration.

**Reply by haikyuu** [reply]
Thanks a lot. I will try it and report back how it goes
